python/dgl/dataloading/shadow.py

```diff
@@ -13,9 +13,6 @@ class ShaDowKHopSampler(object):
     seed nodes from which the neighbors are sampled will appear the first in the
     induced nodes of the subgraph.
```

Comment:
```
    It performs node-wise neighbor sampling and returns the subgraph induced by
    all the sampled nodes. The seed nodes from which the neighbors are sampled
    will appear the first in the induced nodes of the subgraph.
```

Do not talk about MFG here as it's not relevant. Don't understand what "The seed nodes from which the neighbors are sampled will appear the first in the induced nodes of the subgraph." means.

Reply:
The second assert statement of the example in the docstring is what I want to say.